spec/features/admin/enterprises/terms_and_conditions_spec.rb

```diff
@@ -54,7 +54,7 @@ def go_to_business_details
 
         go_to_business_details
         expect(page).to have_selector "a[href*='logo-white.pdf'][target=\"_blank\"]"
-        expect(page).to have_content time.strftime("%F %T %z")
+        expect(page).to have_content time.strftime("%F %T")
```

Comment:
That's a shame. It would be good to verify that we print the time zone. It's important to clarify in countries with multiple time zones.